### ISSUE TITLE
Feature: Update Xcode to 15.3

### ISSUE THREAD

**Issue by mpogue2** [mpogue2/SquareDesk#1060]

**Comment by mpogue2** [comment]
Done.  Commit: 2472f09d08c1b0734304fa54e89688c31a127f3a

QMAKE_MAC_SDK is now macosx14.4 (latest from Apple).

I also put some comments in the .pro file to make it easier to upgrade next time.  The compiler is correctly telling us that the problem was on copydata1dir, but it wasn't clear WHY.  It was because $(MKDIR) fails if the file exists already, AND the copydata1 was getting executed before the copydata1dir.  I changed the .pro file so that it would correctly make the MacOS folder BEFORE trying to stick the sd_calls.dat file there (which makes a thing called "MacOS" which is NOT a directory, it's the .dat binary file).  I confirmed that it all works by blowing away the entire "build...Release" directory, and rebuilding.  No more "file exists" errors now!
